pkg/managed/istio.go

```diff
@@ -173,13 +173,16 @@ func CreateServiceEntries(mbPair *types.ModelBindingPair, availableManagedCluste
 }
 
 func CreateDestinationRules(mbPair *types.ModelBindingPair, availableManagedClusterConnections map[string]*util.ManagedClusterConnection) error {
-	glog.V(6).Infof("Creating/updating DestinationRules for VerrazzanoBinding %s", mbPair.Binding.Name)
-
 	// Parse out the managed clusters that this binding applies to
 	filteredConnections, err := util.GetManagedClustersForVerrazzanoBinding(mbPair, availableManagedClusterConnections)
 	if err != nil {
 		return err
 	}
+	return createDestinationRules(mbPair, filteredConnections)
+}
+
+func createDestinationRules(mbPair *types.ModelBindingPair, filteredConnections map[string]*util.ManagedClusterConnection) error {
```

Comment:
I understand the lowercase c distinguishes the 2 functions but that is subtle and maybe call the private one createDestinationRulesPriv.  Up to you.